clip_retrieval/cli.py

```diff
@@ -17,6 +18,7 @@ def main():
         {
             "back": clip_back,
             "inference": clip_inference,
+            "worker": worker,
```

Comment:
Let's call it inference.worker

Reply:
44090984f49474b672e90317464d1ee2d923feca